kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -198,10 +202,29 @@
     },
     $trNameSpace: 'userPage',
     $trs: {
-      noUsersExist: 'No Users Exist.',
-      allUsersFilteredOut: 'No users match the filter.',
+      // input & accessibility labels
```

Comment:
You could potentially come up with an internal convention. Note that these IDs are strings - I don't think they necessarily need to be fully camel-case.

for example, could do:

```javascript
$trs: {
  'noUsersExist': 'No Users Exist.',
  'allUsersFilteredOut': 'No users match the filter.',
  'input.filterUserType': 'Filter User Type',
  'input.editAccountInfo': 'Edit Account Information',
  'filter.allUsers': 'All Users',
  'filter.admins': 'Admins',
...
```

cc @rtibbles  @aronasorman @radinamatic  maybe there are established conventions?